index.d.ts

```diff
@@ -1,5 +1,26 @@
-class PolyK {
-    static Slice(p: number[][][], ax: number, ay: number, bx: number, by: number)
+declare var PolyK: {
+    Slice(p: number[], ax: number, ay: number, bx: number, by: number): number[],
```

Comment:
Sounds like an ideal solution! As this one got somehow merged, I'll come with new PR during this week.